flixel/FlxG.hx

```diff
@@ -350,6 +351,30 @@ class FlxG
 	}
 	
 	/**
+	 * Returns the first member of Group that overlaps InputObject, or null if none apply.
+	 * 
+	 * @param	InputObject	First object to test against
+	 * @param	Group		The group to test against
+	 * @return	First member of Group intersecting InputObject, or null
+	 */
+	public static function getFirstOverlap(InputObject:FlxBasic, Group:FlxGroup):FlxObject
+	{
+		var minIndex = -1;
+		var firstObject:FlxObject = null;
+		FlxG.overlap(InputObject, Group, function MyOverlap(Objec1:FlxObject, Object2:FlxObject) {
```

Comment:
@MSGhero is it possible without the closure to access "minIndex" and "firstObject"? The only way I see it is to create an inner class with those 2 properties that will be used only in this function, and pass an object of that kind as the third parameter of the  `myOverlap()` function, assuming that the signature allows to get extra params...